### ISSUE TITLE
zigbee-herdsman fails to start on CC1352P-2: Timeout on SRSP - SYS - version

### ISSUE THREAD

**Issue by magnusnordlander** [Koenkk/zigbee2mqtt#2895]
# Bug Report
## What happened
I try to start zigbee2mqtt in Docker on a HPE Microserver with a CC132P-2 using the following configuration: https://gist.github.com/magnusnordlander/c9f8c979f44a308c5a3b5e3e2e3ce1e3#file-configuration-yaml

When running zigbee2mqtt:latest, I get the following log messages: https://gist.github.com/magnusnordlander/c9f8c979f44a308c5a3b5e3e2e3ce1e3#file-log-log

When running zigbee2mqtt:latest-dev, I get the following log messages:
https://gist.github.com/magnusnordlander/c9f8c979f44a308c5a3b5e3e2e3ce1e3#file-latest-dev-log

## What did you expect to happen
zigbee2mqtt should start

## Debug Info
zigbee2mqtt version: 1.9.0 or 1.9.0-dev (commit #558153e)
Firmware version: CC1352P_2_20191106.hex

I have tried running zigbee2mqtt on directly on the host, power cycling the device, reflashing the device (following the steps on https://www.zigbee2mqtt.io/information/flashing_via_uniflash.html), verifying the flashed image, using a different USB port and changing the PAN id. None of this has yielded any results.

It seeems to me that the device just isn't responding whenever commands are being sent. Is there any simple way (e.g. using minicom) that I can use to coax a response out of the device on another machine? E.g., I tried cat:ing the device and in another shell while running `echo -e "\xFE\x00\x21\x02\x23" > /dev/ttyACM1`, just trying to get any response from the firmware. This didn't work, but I don't know if it was even supposed to.

**Comment by CodeFinder2** [comment]
Once connected / powered up, try pressing the button next to the USB connector momentarily.

Then try (re-) starting Z2M.

**Comment by magnusnordlander** [comment]
Wow, that seems to be working; at least it's starting up now. Is that a firmware reset or something? Should I expect to have to do that every time the device is reconnected?

**Comment by CodeFinder2** [comment]
Unfortunately, yes. But it seems to depend on the underlying system/architecture/USB driver/...(?). See more infos here: https://github.com/Koenkk/zigbee2mqtt/issues/2162#issuecomment-568617134 (also in the other posts there).
TL;DR: I have to push that button on my Intel i7 (x86) as well but *not* on my Odroid XU4 (arm).

I guess, this can be closed now? 🙈

**Comment by magnusnordlander** [comment]
Yeah, absolutely. This probably deserves a notice in the FAQ or something though.

**Comment by CodeFinder2** [comment]
Yes I think so, too! According to my knowledge, you just have to create a pull request against https://github.com/Koenkk/zigbee2mqtt.io/blob/master/docs/information/FAQ.md (@Koenkk  please correct me if I am wrong, I couldn't find any specific information on that in your docs.)

**Comment by Koenkk** [comment]
It should definitely be added to the docs, can you create a similar section of https://github.com/Koenkk/zigbee2mqtt.io/blob/master/docs/information/FAQ.md#press-the-reset-button-on-the-device-only-applicable-for-cc2531 above it?

**Comment by CodeFinder2** [comment]
> It should definitely be added to the docs, can you create a similar section of https://github.com/Koenkk/zigbee2mqtt.io/blob/master/docs/information/FAQ.md#press-the-reset-button-on-the-device-only-applicable-for-cc2531 above it?

Done!

**Comment by dvbit** [comment]
Hi I tried and verified all that is said in github and faq to no avail.
device is linked to correct ACM
port is writable 
user is in diallout group
permissions are ok
pushed the reset button

always get the 600 ms error.

Running on a dell x86 6th gen i5 portable

any clue or suggestion?

**Comment by CodeFinder2** [comment]
@dvbit did you try to reflash?

**Comment by dvbit** [comment]
Reflash with the same image? 
The module was fresh out of the box
I erased  the flash using uniflash on debian
Loaded zstack 3 im age and verified(ok)
Should reflashing with same image hlp?
I will try. Tx

**Comment by dvbit** [comment]
No.i will . 
How can this help? Clears some areas?

**Comment by dvbit** [comment]
Flashed : no success

**Comment by dvbit** [comment]
trying on a raspberrry pi 2. I read somewhere certain revision of the board will not work. Where do I read that info?

**Comment by dvbit** [comment]
Ok. No success on the raspberry too.
Qn: do other zigbee network interfere? I have Xiaomi and Hue on and (likely) also the amz echo although without attacched device on

**Comment by dvbit** [comment]
Ok. On Raspberry now works. setting the serial in configuration to auto detect
